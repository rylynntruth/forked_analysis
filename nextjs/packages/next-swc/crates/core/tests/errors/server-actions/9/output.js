import 'my-module';
'use server';
