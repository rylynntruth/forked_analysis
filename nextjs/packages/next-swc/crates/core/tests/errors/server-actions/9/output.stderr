
  x The "use server" directive must be at the top of the file, and cannot be wrapped in parentheses.
   ,-[input.js:1:1]
 1 | import 'my-module'
 2 | ;('use server')
   :  ^^^^^^^^^^^^^^
   `----
